data-prepper-plugins/s3-source/src/main/java/com/amazon/dataprepper/plugins/source/S3ObjectWorker.java

```diff
@@ -34,6 +34,8 @@ class S3ObjectWorker {
     static final String S3_OBJECTS_FAILED_METRIC_NAME = "s3ObjectsFailed";
     static final String S3_OBJECTS_SUCCEEDED_METRIC_NAME = "s3ObjectsSucceeded";
     static final String S3_OBJECTS_TIME_ELAPSED_METRIC_NAME = "s3ObjectReadTimeElapsed";
+    public static final String BUCKET_FIELD_NAME = "bucket";
+    public static final String KEY_FIELD_NAME = "key";
```

Comment:
These aren't used outside of this class and can be marked as `private`

Reply:
Oh IntelliJ!

Thanks for catching that.